### PR TITLE
Fix _changes?filter=_design

### PR DESCRIPTION
**Pull request by bessbd**
## Overview
`_changes?filter=_design` was returning a 
```
{
  "error": "unknown_error",
  "reason": "function_clause",
  "ref": 240731020
}
```

This PR fixes that and re-enabled the related elixir test.

## Testing recommendations

`mix test --trace test/elixir/test/changes_test.exs:220`

## Checklist

- [ ] Code is written and works correctly
- [ ] Changes are covered by tests
- [ ] Any new configurable parameters are documented in `rel/overlay/etc/default.ini`
- [ ] A PR for documentation changes has been made in https://github.com/apache/couchdb-documentation

### CODE REVIEW

File: src/chttpd/src/chttpd_changes.erl [hunk 1]
```diff
@@ -466,7 +466,12 @@ send_changes_doc_ids(Db, StartSeq, Dir, Fun, Acc0, {doc_ids, _Style, DocIds}) ->
 
 
 send_changes_design_docs(Db, StartSeq, Dir, Fun, Acc0, {design_docs, _Style}) ->
-    FoldFun = fun(FDI, Acc) -> {ok, [FDI | Acc]} end,
+    FoldFun = fun(FDI, Acc) ->
+        case FDI of
+            {row, Row} -> {ok, [fabric2_db:get_full_doc_info(Db, proplists:get_value(id, Row)) | Acc]};
```
Comment (nickva) [inline src/chttpd/src/chttpd_changes.erl:471]: Let's make sure the line is shorter than 80 columns if possible.

**Review by nickva — APPROVED** [review]
+1

But see the minor nit about line length

File: src/chttpd/src/chttpd_changes.erl (hunk 1 above)
Comment (bessbd) [reply src/chttpd/src/chttpd_changes.erl]: Done in 343015774

**Comment by bessbd** [comment]
Thank you for the review, @nickva ! I'm about to merge this.

**Pull request merged**